src/coreclr/tools/aot/ILCompiler.LLVM/Compiler/LLVMCodegenCompilation.cs

```diff
@@ -60,6 +65,8 @@ public sealed class LLVMCodegenCompilation : RyuJitCompilation
             DIBuilder = Module.CreateDIBuilder();
             DebugMetadataMap = new Dictionary<string, DebugMetadata>();
             ILImporter.Context = Module.Context;
+            LLVMContextSetOpaquePointers(Module.Context, false);
```

Comment:
I am curious how this works linking wise. Does the Jit module still produce declarations/definitions of functions using precise pointer types, or...?

Reply:
In the Jit module 
```
declare i8 @S_P_CoreLib_System_Runtime_CompilerServices_RuntimeHelpers__Equals(ptr)
```
In the IL module
```
define i8 @S_P_CoreLib_System_Runtime_CompilerServices_RuntimeHelpers__Equals(i8* %0)
```
This is used in `Object__Equals` from the Jit module.  You can run wasm2wat on the final wasm and you see:

```
  (func $Object__Equals (type 1) (param i32) (result i32)
    (local i32 i32 i32 i32 i32 i32 i32 i32 i32 i32 i32 i32 i32)
    local.get 0
    i32.load
    local.set 1
    i32.const 4
    local.set 2
    local.get 0
    local.get 2
    i32.add
    local.set 3
    local.get 3
    i32.load
    local.set 4
    i32.const 8
    local.set 5
    local.get 0
    local.get 5
    i32.add
    local.set 6
    i32.const 8
    local.set 7
    local.get 0
    local.get 7
    i32.add
    local.set 8
    local.get 8
    local.get 1
    i32.store
    i32.const 12
    local.set 9
    local.get 0
    local.get 9
    i32.add
    local.set 10
    local.get 10
    local.get 4
    i32.store
    local.get 6
    call $S_P_CoreLib_System_Runtime_CompilerServices_RuntimeHelpers__Equals
```
As Wasm just uses i32 for the pointers, I guess the linker doesn't really care.

Reply:
I see. Presumably means we can drop all of `GetParameterType`/`GetArgTypeIncludingParameterized` (in a follow up change) as well.

Reply:
The wat for `$S_P_CoreLib_System_Runtime_CompilerServices_RuntimeHelpers__Equals`
```
func $S_P_CoreLib_System_Runtime_CompilerServices_RuntimeHelpers__Equals (type 1) (param i32) (result i32)
```
Where `type 1 ` is
```
  (type (;1;) (func (param i32) (result i32)))
```